### PR TITLE
Fix #456: [FR] Handle ar-mot and ar-terme

### LINKED ISSUE #456: [FR] Handle ar-mot and ar-terme
- Wiktionary page: https://fr.wiktionary.org/wiki/alezan
https://fr.wiktionary.org/wiki/Muad-Dib

Wikicode:
```
{{ar-mot|elHasan_}}
```

Output:
```
<i>(Ar-mot)</i>
```

Expected:
```
 الحَسَن (elHasan_) 
```

---

Model link, if any: https://fr.wiktionary.org/wiki/Mod%C3%A8le:ar-mot

### PR DESCRIPTION
**Pull request by lasconic**
- [x] Fixes #456
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

File: scripts/lang/fr/__init__.py [hunk 1]
```diff
@@ -697,6 +698,9 @@ def last_template_handler(
         >>> last_template_handler(["Légifrance", "base=CPP", "numéro=230-45"], "fr")
         ''
 
+        >>> last_template_handler(["ar-mot", "elHasan_"], "fr")
+        '<span style="line-height: 0px;"><big><big>الحَسَن</big></big></span> <small>(elHasan_)</small>'
```
Comment (BoboTiG) [inline scripts/lang/fr/__init__.py:702]: The `<big>` tag is deprecated and no more used in HTML5, cf https://developer.mozilla.org/en-US/docs/Web/HTML/Element/big. Should we use something like `<span style="font-size:larger">...</span>"`?

File: scripts/lang/fr/arabiser.py [hunk 2]
```diff
@@ -0,0 +1,579 @@
...
+                if en_arabe.get(a_traiter):
+                    transcription += en_arabe[a_traiter]
+                # cas d'une consonne en fin de mot - rajouter un sukun final
+                if texte[curseur + 1] == " ":
+                    transcription += en_arabe["°"]
+
+    transcription = "".join(
+        [c for c in transcription if unicodedata.name(c) != "DOTTED CIRCLE"]
+    )
+    # print([unicodedata.name(c) for c in transcription])
```
Comment (BoboTiG) [inline scripts/lang/fr/arabiser.py:575]: You could remove debug lines.

**Comment by lasconic** [comment]
FYI I didn't test this on kobo at all...

File: scripts/lang/fr/arabiser.py (hunk 2 above)
Comment (lasconic) [reply scripts/lang/fr/arabiser.py]: Gone

File: scripts/lang/fr/__init__.py (hunk 1 above)
Comment (lasconic) [reply scripts/lang/fr/__init__.py]: big is gone. Again, no idea how this span will look on a device.

**Comment by BoboTiG** [comment]
I will be able to test and take a screenshot.

**Comment by BoboTiG** [comment]
Sorry for the dealy, I will test it tomorrow for sure :)

**Pull request merged**